regolith/helpers/l_generalhelper.py

```diff
@@ -14,19 +14,22 @@
 def subparser(subpi):
     subpi.add_argument(
         "coll",
-        help='collection where the lister is going to run.')
+        help='Collection where the lister is going to run.')
     subpi.add_argument(
         "-f","--kv_filter",
         nargs="+",
-        help='search the given collection by key element pairs')
+        help="Search the given collection by key element pairs. "
+             "e.g. < -f name simon >, will return the id of all the people "
+             "who's name contains simon.")
     subpi.add_argument(
         "-r",  "--return_fields",
         nargs="+",
-        help='keys of fields whose values will be printed after filtering.')
+        help="Specify what keys to return values from when running --filter. "
```

Comment:
I think this should work independent of whether -f is specified?  Why not?  What if I want to get certian key values but not apply a filter so from everything in the collection?  It seems like a reasonable request, right?

wording -> "Specify from which keys to print values, e.g.,' regolith helper lister people -r name status title"

Reply:
done!